### PR TITLE
Add Oct 2

### PR DESCRIPTION
**Pull request by blackboxsw**

### CODE REVIEW

File: doc/2018/2018-10-02.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+and use the Ubuntu Server Team [mailing list](https://lists.ubuntu.com/mailman/listinfo/ubuntu-server).
+
+## Spotlight: Cosmic Release Imminent and New Core Developer
+Cosmic is in its [final week](https://wiki.ubuntu.com/CosmicCuttlefish/ReleaseSchedule) of testing. Next week the final freeze will occur and release candidate get published. If you are interested head over to the [daily ISO page](http://cdimage.ubuntu.com/ubuntu-server/daily/current/) and give Cosmic a spin! Our developers are working to close out any sigificant bugs and documentation to add a last bit of polish to the release which is scheduled for Oct 11th.
+
+Also a great cause for celebration in this issue is that Ubuntu Server Team's [Andreas Hasenack](https://wiki.ubuntu.com/AndreasHasenack/CoredevApplication) earned a spot as an [Ubuntu Core Developer](https://wiki.ubuntu.com/UbuntuDevelopers#Ubuntu_Core_Developers). His tenure maintaining crucial ubuntu packages, community involvement, broad technical knowledge and eye toward quality has brought an additional level of excellence to Ubuntu.  We are excited he now has 'keys to the kingdom', allowing him to update and maintain any package in Ubuntu.
```
Comment (powersj) [inline doc/2018/2018-10-02.md]: Should it be "Ubuntu packages"? captial U

File: doc/2018/2018-10-02.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
+# Hello Ubuntu Server
+
+The purpose of this communication is to provide a status update and
+highlights for any interesting subjects from the Ubuntu Server Team. If
+you would like to reach the server team, you can find us at
+the #ubuntu-server channel on Freenode. Alternatively, you can sign up
+and use the Ubuntu Server Team [mailing list](https://lists.ubuntu.com/mailman/listinfo/ubuntu-server).
+
+## Spotlight: Cosmic Release Imminent and New Core Developer
+Cosmic is in its [final week](https://wiki.ubuntu.com/CosmicCuttlefish/ReleaseSchedule) of testing. Next week the final freeze will occur and release candidate get published. If you are interested head over to the [daily ISO page](http://cdimage.ubuntu.com/ubuntu-server/daily/current/) and give Cosmic a spin! Our developers are working to close out any sigificant bugs and documentation to add a last bit of polish to the release which is scheduled for Oct 11th.
```
Comment (powersj) [inline doc/2018/2018-10-02.md]: Final weeks, not week. Release is Oct 18, not the 11th.

"and documentation to add a last bit of polish to the release which is scheduled for Oct 11th."

This does not read smoothly.

**Review by powersj — CHANGES_REQUESTED** [review]

File: doc/2018/2018-10-02.md (hunk 1 above)
Comment (blackboxsw) [reply doc/2018/2018-10-02.md]: done thx.

File: doc/2018/2018-10-02.md (hunk 2 above)
Comment (blackboxsw) [reply doc/2018/2018-10-02.md]: reworded.

**Pull request merged**